tests/unit/components/App.spec.ts

```diff
@@ -9,6 +9,8 @@ import { ENTITY_INIT } from '@/store/entity/actionTypes';
 import { LANGUAGE_PREFERENCE } from '@/store/user/actionTypes';
 import { EDIT_LINK_URL_INIT } from '@/store/links/actionTypes';
 
+config.logModifiedComponents = false;
```

Comment:
This has to do with vue-test-utils and vue-class-component. To be honest, I'm a little confused why this error pops up in this file, which only `shallowMount`s components. Any ideas are welcome. Without this, we get the following warning:

>console.error node_modules/@vue/test-utils/dist/vue-test-utils.js:15
[vue-test-utils]: The child component <EditPen> has been modified to ensure it is created with properties injected by Vue Test Utils. 
This is because the component was created with Vue.extend, or uses the Vue Class Component decorator. 
Because the component has been modified, it is not possible to find it with a component selector. To find the component, you must stub it manually using the stubs mounting option, or use a name or ref selector. 
You can hide this warning by setting the Vue Test Utils config.logModifiedComponents option to false.

Reply:
Ermahgerd, it does!!